inspirehep/bat/pages/create_literature.py

```diff
@@ -70,6 +70,64 @@ def _submit_thesis():
     return ArsenicResponse(_submit_thesis)
 
 
+def submit_book(input_data):
+    def _submit_book():
+        return (
+            'The INSPIRE staff will review it and your changes will be added '
+            'to INSPIRE.'
+        ) in WebDriverWait(Arsenic(), 10).until(
+            EC.visibility_of_element_located(
+                (
+                    By.XPATH,
+                    '(//div[@class="alert alert-success alert-form-success"])',
+                )
+            )
+        ).text
```

Comment:
This will have to modified when merging https://github.com/inspirehep/inspire-next/pull/2517.